web/client/themes/default/less/geostory.less

```diff
@@ -310,7 +314,8 @@
         &.ms-size-medium .ms-media { width: @ms-story-size-medium; }
         &.ms-size-small .ms-media { width: @ms-story-size-medium; }
     }
-    .ms-section-title .ms-content-text {
+    .ms-section-title .ms-content-text,
+    .ms-section-banner .ms-content-text {
```

Comment:
I think this style is not needed because it applies to content style` .ms-content-text`